### DISCUSSION TITLE
Dúvida no codigo do php

### DISCUSSION THREAD

**Discussion by Paulo23k** [Paulo23k/OOP-DEV#4] [Q&A] [answered]
$result = array_reduce(array_map(fn($x) => strlen($x) % 2 == 0 ? strtoupper($x) : strrev($x), explode("

**Comment by Libann23k** [comment, marked as answer]
O código transforma cada palavra de uma string, convertendo para maiúsculas se tiver tamanho par ou invertendo se ímpar, e concatena os dois primeiros caracteres de cada palavra transformada em uma única string.
